api/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.12
```

Comment:
Can leave these in for now but should be overwritten when pushing the api-test branch to main.